### ISSUE TITLE
Remove CacheURL Delivery Service Field

### ISSUE THREAD

**Issue by jhg03a** [apache/trafficcontrol#3225]
CacheURL was a plugin that was deprecated in ATS6 and removed in ATS7.  ATS6 is now dead.  We should remove all traces of cacheURL from the TO API, TODB Database, Native-client API Libraries, & TP Web UI.

This is previously discussed in #2957 and #1130

**Comment by jhg03a** [comment]
Consensus appears to have been reached and a deprecation notice issued for the 3.x release.

https://lists.apache.org/thread.html/d0a8805315819b5e4d9d9bf7f19bfd2b895924dc4247bec299b3ba15@%3Cdev.trafficcontrol.apache.org%3E

**Comment by shamrickus** [comment]
Working on this, see #4644

**Comment by mitchell852** [comment]
i feel like this should be done by now with 4 merged PRs :)

is it @shamrickus ?

**Comment by shamrickus** [comment]
No. It's now officially deprecated, which it wasn't before. We were returning the field in api v3 so until that's gone we can't get rid of it completely

**Comment by mitchell852** [comment]
> No. It's now officially deprecated, which it wasn't before. We were returning the field in api v3 so until that's gone we can't get rid of it completely

gotcha. thanks for the clarification. hard to get rid of these things, huh?
